core/runtime/jsruntime.rs

```diff
@@ -1942,8 +1962,13 @@ impl JsRuntime {
   /// Implementors must manually call [`JsRuntime::run_event_loop`] to drive
   /// module evaluation future.
   ///
-  /// `Error` can usually be downcast to `JsError` and should be awaited and
-  /// checked after [`JsRuntime::run_event_loop`] completion.
+  /// Rejection errors from module evaluation are treated as unhandled rejections, and
```

Comment:
Ditto about "rejection errors". Is it on purpose that this comment is duplicated?

Reply:
Reworked this comment.. Let me know what you think